### DISCUSSION TITLE
Race / Event Mode

### DISCUSSION THREAD

**Discussion by stuartlynne** [SauceLLC/sauce4zwift#14] [General]
Can you prevent other competitors' data from being seen when in a race or event?

Other than position. 

This may be required to keep sauce available in some races.

**Comment by mayfield** [comment]
Anythings possible.  If that's how race organizers want to proceed I'm happy to accommodate.

**Reply by stuartlynne** [reply]
Might be a good idea to be proactive. Seems like there are some conversations happening on various lists, e.g. WTRL Team Managers list. With the usual warnings and read the TOS etc.

**Reply by suskozaver** [reply]
besides the fact, it shows real-time data, it doesn't show much more than Zwiftpower provides in LIVE section?
